### PR TITLE
show error in case of a missing column in the data

### PR DESCRIPTION
**Pull request by kenseii**
This PR implements the error handling in case there are missing columns in the gtfs datasets.

### CODE REVIEW

File: gtfs_go_loader.py [hunk 1]
```diff
@@ -127,6 +137,20 @@ def canceled(self):
         self.loadingAborted.emit()
         self.close()
 
+    def raised_error(self, error):
```
Comment (Kanahiro) [inline gtfs_go_loader.py:140]: In my opinion, this method should be named as meaning it's function or timing when are triggered.

example:  ``show_error_message()`` or ``on_raised_error()``

File: gtfs_go_loader.py [hunk 2]
```diff
@@ -33,6 +33,9 @@
 from qgis.PyQt import QtWidgets, uic
 from qgis.PyQt.QtCore import QThread, pyqtSignal
 
+from qgis.utils import iface
```
Comment (Kanahiro) [inline gtfs_go_loader.py:36]: I haven't known it is possible to get a ref to QgsInterface by such a simple code, thank you!

**Review by Kanahiro — COMMENTED** [review]
Thanks you for good codes! I added some trivial comments, please confirm it.

File: gtfs_go_loader.py (hunk 1 above)
Comment (kenseii) [reply gtfs_go_loader.py]: Thank you, i fixed it

**Review by Kanahiro — APPROVED** [review]
LGTM!

**Pull request merged**